File: KDT_class/Day02_Basic/src/Quiz01.java
public class Quiz01 {
    public static void main(String[] args) {
        byte b = 10;
        short s = 20;
        char c = 'A';
        int i;
        float f;
        long l = 100L;

        s = b;
        c = (char)b;
        s = (short)c;
        c = (char)s;
        i = (int)100L;
        l = 500;
        f = l;
        f = 5.11f;
    }
}

File: KDT_class_phaser/Quiz/Quiz01.js
class Quiz01 extends Phaser.Scene {
    constructor() {
        super({key: 'Quiz01'});
    }

    preload() {
        this.load.image('background', 'Assets/sideScrollingBackground.jpg');
    }

    create() {
        this.background = this.add.tileSprite(this.cameras.main.centerX, this.cameras.main.centerY, 1000, 360, 'background');
    }

    update() {
        this.background.tilePositionX += 3;
    }
}